bdai_ros2_wrappers/bdai_ros2_wrappers/single_goal_action_server.py

```diff
@@ -9,14 +9,13 @@
 from bdai_ros2_wrappers.type_hints import ActionType
 
 # Note: for this to work correctly you must use a multi-threaded executor when spinning the node!  E.g.:
```

Comment:
@amessing-bdai nit: maybe drop the trailing `e.g.`?
```suggestion
# Note: for this to work correctly you must use a multi-threaded executor when spinning the node!
```

Reply:
Fixed.